pkgs/development/tools/misc/docopts/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "docopts";
+  version = "0.6.3-rc2";
+
+  src = fetchFromGitHub {
+    owner = "docopt";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-PmsTkPT/sf70MKYLhHvjCDb2o3VQ1k7d++RUW7rcoAg=";
+  };
+
+  goPackagePath = "github.com/docopt/${pname}";
+
+  goDeps = ./deps.nix;
+
+  postInstall = ''
+    install -D -m 755 ./go/src/$goPackagePath/docopts.sh $out/bin/docopts.sh
+    rm $out/bin/json_t
```

Comment:
Please only build the binaries we need. I think you can achieve this with:

```
subPackages = [ "docopt" ];
```

Reply:
Done.

Btw. you suggestion just gave me _'cannot find package "github.com/docopt/docopts/docopts"', I used ` subPackages = [ "./" ];`.